manifests/prometheus/alerts.d/diego_cell_available_memory.yml

```diff
@@ -5,27 +5,34 @@
   value:
     name: BoshDiegoCellAvailableMemory
     rules:
-      - record: bosh_job:mem_percent:diego_cell:avg5m
-        expr: 100 - avg_over_time(bosh_job_mem_percent{environment="((metrics_environment))",bosh_job_name="diego-cell"}[5m])
+      - record: bosh_job_mem_percent_free
+        expr: 100 - bosh_job_mem_percent
+
       - &alert
         alert: BoshDiegoCellAvailableMemory_Warning
-        expr: bosh_job:mem_percent:diego_cell:avg5m < 55
+        expr: >-
+          avg(bosh_job_mem_percent_free{bosh_job_name="diego-cell"})
+          by (bosh_job_name,bosh_deployment,bosh_name,environment,job)
+          < 55
         for: 2h
         labels:
           severity: warning
           notify: email
         annotations:
-          summary: Low memory free on {{ $labels.bosh_job_name }}.
+          summary: Low memory free on {{ $labels.bosh_job_name }}
           description: >
-            {{ $labels.bosh_job_name }}/{{ $labels.bosh_job_id }}
+            {{ $labels.bosh_job_name }}
             memory free was low {{ $value | printf "%.0f" }}%
             in the last two hours on average.
             Review if we need to scale...
           url: https://team-manual.cloud.service.gov.uk/architecture_decision_records/ADR021-cell-capacity-assignment-2/#decision
 
       - <<: *alert
         alert: BoshDiegoCellAvailableMemory_Critical
-        expr: bosh_job:mem_percent:diego_cell:avg5m < 50
+        expr: >-
```

Comment:
this should be similarly replaced: `avg(bosh_job_mem_percent{bosh_job_name="diego-cell"}) >= 50`